### PR TITLE
Include c8y cli

### PR DESCRIPTION
**Pull request by neuerburg**
This pull requests adds a reference to the C8Y-Go-CLI to our welcome developer documentation as well as the additional resources section.

### CODE REVIEW

File: content/additional-resources/more-documentation.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ sector:
 
 A complete OpenAPI Specification for the {{< product-c8y-iot >}} REST API is available at [https://{{< domain-c8y >}}/api/core/](https://{{< domain-c8y >}}/api/core). The OpenAPI Specification describes the {{< product-c8y-iot >}} REST API (that is, available endpoints, operations on each endpoint, input and output for each operation, authentication methods and more) following the OpenAPI 3.0 standard.
 
+
+### Cumulocity CLI
```
Comment (BeateRixen) [inline content/additional-resources/more-documentation.md:14]:
```suggestion
### {{< product-c8y-iot >}} CLI {#cumulocity-iot-cli}
```

File: content/additional-resources/more-documentation.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ sector:
 
 A complete OpenAPI Specification for the {{< product-c8y-iot >}} REST API is available at [https://{{< domain-c8y >}}/api/core/](https://{{< domain-c8y >}}/api/core). The OpenAPI Specification describes the {{< product-c8y-iot >}} REST API (that is, available endpoints, operations on each endpoint, input and output for each operation, authentication methods and more) following the OpenAPI 3.0 standard.
 
+
+### {{< product-c8y-iot >}} CLI {#cumulocity-iot-cli}
+
+To simplify the interaction with the platform for developers, the Cumulocity IoT CLI tool provides a powerful command-line interface. This tool allows you to manage devices, applications, and other platform services efficiently through terminal commands, streamlining development and automation tasks. To get started, visit the [CLI tool documentation](https://goc8ycli.netlify.app/docs/introduction/) for detailed instructions and examples.
```
Comment (BeateRixen) [inline content/additional-resources/more-documentation.md:16]:
```suggestion
To simplify the interaction with the platform for developers, the {{< product-c8y-iot >}} CLI tool provides a powerful command-line interface. This tool allows you to manage devices, applications, and other platform services efficiently through terminal commands, streamlining development and automation tasks. To get started, visit the [CLI tool documentation](https://goc8ycli.netlify.app/docs/introduction/) for detailed instructions and examples.
```

File: content/welcome-developers/developer-topics.md [hunk 3]
```diff
@@ -33,6 +33,11 @@ Check out the following sections if you are looking for step-by-step instruction
 
 A complete OpenAPI Specification for the {{< product-c8y-iot >}} REST API is available at [https://{{< domain-c8y >}}/api/core](https://{{< domain-c8y >}}/api/core). The {{< openapi >}} describes the entire {{< product-c8y-iot >}} REST API (that is, available endpoints, operations on each endpoint, input and output for each operation, authentication methods and more) following the OpenAPI 3.0 standard.
 
+### Cumulocity CLI
```
Comment (BeateRixen) [inline content/welcome-developers/developer-topics.md:36]:
```suggestion
### {{< product-c8y-iot >}} CLI {#cumulocity-iot-cli}
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@neuerburg Shall I merge this?

**Comment by neuerburg** [comment]
@BeateRixen , yes if you have reviewed the changes please go ahead with merging!

**Pull request merged**